lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -1507,7 +1507,7 @@ class LineChartPainter extends AxisChartPainter<LineChartData> {
 
     /// it holds list of nearest touched spots of each line
     /// and we use it to draw touch stuff on them
-    final touchedSpots = <LineBarSpot>[];
+    final touchedSpots = <TouchLineBarSpot>[];
```

Comment:
We need to update the return type of this function from `List<LineBarSpot>` to `List<TouchLineBarSpot>`.

It's a breaking change and we need to notify our users.

Reply:
It would be great to add these changes to our [CHANGELOG.md](https://github.com/imaNNeoFighT/fl_chart/blob/dev/CHANGELOG.md).

I should update our [CONTRIBUTING.md](https://github.com/imaNNeoFighT/fl_chart/blob/master/CONTRIBUTING.md) guideline to explain how CHANGELOG.md works. But it's straightforward. You can check our previous versions changelog.

Reply:
I added changed the type in `LineTouchResponse` and added it to the changelog